tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -225,6 +245,57 @@ def test_file_no_stack_root_match(self, mock_integration):
         assert response.data["error"] == "stack_root_mismatch"
         assert response.data["integrations"] == [serialized_integration(self.integration)]
 
+    @mock.patch.object(ExampleIntegration, "get_blame_for_file")
```

Comment:
Could you please create a new class to group all tests that exercise the codecov related functionality?